doc/getting_started/index.rst

```diff
@@ -137,7 +137,12 @@ We'll also install Zephyr's multi-purpose west tool.
 
          .. code-block:: bash
 
-            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+            brew install cmake ninja gperf ccache dfu-util qemu dtc python3 libgit2
```

Comment:
Hi All,
I would strongly recommend that someone test this on a fresh OS that this change is not required. I am quite certain that this change was needed with the NCS version I pulled in January, but it could be that dependency was a mistake and might have been removed by some other fix. I do not have a MacOS at home, else I would have tested this. This change should be here unless someone confirms/test that this is not needed.

Reply:
> I am quite certain that this change was needed with the NCS version I pulled in January [...] This change should be here unless someone confirms/test that this is not needed.


I don't really agree. This is zephyr, not NCS. I think the burden is on the submitter to prove that the change is needed for Zephyr.